Models/SupabaseDatabase.cs

```diff
@@ -353,24 +354,32 @@ public async Task<List<ActiveQuiz>> GetActiveQuizzesByActiveQuizIds(List<long> a
         }
     }
 
-    public async Task<bool> SubmitMultipleChoiceQuestionAnswer(ActiveQuestion question, int choice) {
+    public async Task<bool> SubmitMultipleChoiceQuestionAnswer(ActiveQuestion question, int[]? choices) {
         try {
+
             await Client // Insert into the response table according to the choice selected
+
                 .From<Response>()
                 .Insert(new Response() {
                     ActiveQuizId = question.ActiveQuizId,
                     UserId = UserId,
                     QuestionNo = question.QuestionNo,
+
                     MultipleChoiceResponse = [choice],
```

Comment:
In this method, we no longer have choice, now we have choices. Should this be changed from '[choice]' to 'choices'